source/dataman/export/AMExporterGeneralAscii.cpp

```diff
@@ -599,6 +626,22 @@ bool AMExporterGeneralAscii::writeSeparateFiles(const QString& destinationFolder
 			QFile file;
 			QString separateFileName = parseKeywordString( destinationFolderPath % "/" % option_->separateSectionFileName() );
 
+			bool detectedNonprintable = false;
```

Comment:
Is there a reason why you're not just using the function you made here?  They look pretty similar to me.